src/app/plugins/plugin_colorcalib.cpp

```diff
@@ -21,13 +21,23 @@
 #include "plugin_colorcalib.h"
 #include <QStackedWidget>
 
+#define LUT_COPY_PLACEHOLDER    "(click copy to refresh)"
```

Comment:
Correct, but this is used in several places, which requires the exact string and is likely safer for a single modification point.